src/modules/temperature.cpp

```diff
@@ -53,20 +53,24 @@ namespace modules {
   bool temperature_module::update() {
     m_temp = std::strtol(file_util::contents(m_path).c_str(), nullptr, 10) / 1000.0f + 0.5f;
     int temp_f = floor(((1.8 * m_temp) + 32) + 0.5);
+    int temp_k = m_temp + 273;
```

Comment:
Should be 273.15

Reply:
no, m_temp and temp_k are integers.
the result would be exactly the same

Reply:
My bad, I assumed it's float because we do some floating point operations on the original temperature readings.

I'm reviewing on my phone so I don't have all the context. Will continue reviewing once I'm at my computer again in a week or so, otherwise I'll make more of these mistakes 😅

But I do think we should try to make an accurate conversion, we do in theory have precise enough temperatures for it to matter